app/gui2/scripts/downloadFonts.js

```diff
@@ -39,51 +39,80 @@ function get(options, callback) {
   })
 }
 
-console.info('Downloading Enso font...')
-await fs.rm('./public/font-enso/', { recursive: true, force: true })
-await fs.mkdir('./public/font-enso/', { recursive: true })
-await new Promise((resolve, reject) => {
-  get(ENSO_FONT_URL, (response) => {
-    response.pipe(
-      tar.extract({
-        cwd: './public/font-enso/',
-        strip: 1,
-        filter(path) {
-          // Reject files starting with `.`.
-          return !/[\\/][.]/.test(path)
-        },
-      }),
-    )
-    response.on('end', resolve)
-    response.on('error', reject)
+try {
+  for (const weight of [
+    'Thin',
+    'ExtraLight',
+    'Light',
+    'Regular',
+    'Medium',
+    'SemiBold',
+    'Bold',
+    'ExtraBold',
+    'Black',
+  ]) {
+    await fs.access(`./public/font-enso/Enso-${weight}.ttf`)
+  }
+  console.info('Enso font already downloaded, skipping...')
+} catch {
```

Comment:
Can we check what is the exact error (is it "File not found"?) If it is another (like permission error) it's likely that downloading avail nothing.

Reply:
Pushed after some testing, do feel free to take a look at the latest commit though just in case I've missed something